src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -724,3 +724,54 @@ def test_wait_for_event_repeated_capabilities(session, channels):
     channels_session = session.channels[channels]
     with channels_session.initiate():
         channels_session.wait_for_event(nidcpower.Event.SOURCE_COMPLETE)
+
+
+@pytest.mark.resource_name("4190/0")
+@pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
+@pytest.mark.parametrize(
+    "compensation_function",
+    [
+        nidcpower.Session.perform_lcr_open_custom_cable_compensation,
+        nidcpower.Session.perform_lcr_short_custom_cable_compensation,
+    ],
+)
+def test_perform_lcr_open_short_custom_cable_compensation(session, compensation_function):
+    compensation_function(session)
+
+
+@pytest.mark.resource_name("4190/0")
+@pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
+def test_lcr_custom_cable_compensation_data(session):
+    compensation_data = session.get_lcr_custom_cable_compensation_data()
+    session.configure_lcr_custom_cable_compensation(compensation_data)
+
+    session.configure_lcr_custom_cable_compensation(list(compensation_data))
+
+    session.configure_lcr_custom_cable_compensation(bytes(compensation_data))
+
+    with tempfile.NamedTemporaryFile(suffix='.bin', delete=False) as temp_file:
+        temp_file.write(compensation_data)
+    with open(temp_file.name, 'rb') as reopened_temp_file:
+        compensation_data_bytes_from_file = reopened_temp_file.read()
+    session.configure_lcr_custom_cable_compensation(compensation_data_bytes_from_file)
+
+
+@pytest.mark.resource_name("4190/0")
+@pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
+@pytest.mark.parametrize(
+    "compensation_type",
+    [
+        nidcpower.LCRCompensationType.OPEN_CUSTOM_CABLE,
+        nidcpower.LCRCompensationType.SHORT_CUSTOM_CABLE,
+        nidcpower.LCRCompensationType.OPEN,
+        nidcpower.LCRCompensationType.SHORT,
+        nidcpower.LCRCompensationType.LOAD,
+    ],
+)
+def test_get_lcr_compensation_last_date_and_time(session, compensation_type):
+    last_compensation_datetime = session.get_lcr_compensation_last_date_and_time(compensation_type)
+    assert last_compensation_datetime.year == 1940
```

Comment:
This is a bit useless, since the same odd date is returned regardless of compensation_type.
How do you feel about changing the driver to return different dates based on compensation_type? It would require a note here about the tests beginning to fail with next NI-DCPower runtime.

@sbethur what do you think?

Reply:
Personally, I feel that returning different dates based on compensation_type may not be much useful to the user outside of this test, since the returned dates aren't really documented anywhere in the official NI-DCPower docs (as far as I'm aware), and they are quite arbitrary too. 

Nevertheless, I'm open to making this change in the driver as it's a relatively trivial task.

Reply:
> How do you feel about changing the driver to return different dates based on compensation_type?

This would essentially be validating we got the `compensation_type` enum values right, which we don't typically do with system tests (we rely on metadata review). I say we leave the test as is.

Reply:
It would validate that the customer value for the type makes it to the library correctly.
Ok dropping.